MyBus/MyBus/AppDelegate.swift

```diff
@@ -22,6 +22,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool
     {
         Fabric.with([Crashlytics.self, MGLAccountManager.self])
+        
+        //MGLAccountManager.setAccessToken("pk.eyJ1Ijoibm9zb3VsODgiLCJhIjoiY2lteGt2dHhsMDNrNXZxbHU0M29mcHZnZiJ9.MMbmK9GfcdhpDw2siu0wuA")
```

Comment:
Yes, we are to set Mapbox's token by code in the future in order to obfuscate it.